src/sentry/incidents/logic.py

```diff
@@ -635,6 +649,26 @@ def snapshot_alert_rule(alert_rule):
     )
 
 
+def convert_alert_rule_to_snuba_query(alert_rule):
```

Comment:
Yeah, this will merge first and I wanted data to remain consistent on updates beforehand. I think you're right that it's not strictly necessary since the data should be cleaned up by the migration, but it doesn't hurt and I'd prefer to be cautious.